pyrail/models.py

```
@@ -19,20 +20,114 @@
 
 @dataclass
 class StationsApiResponse(ApiResponse):
     """Holds a list of station objects returned by the 'stations' endpoint."""
 
-    stations: List[Station] = field(metadata=field_options(
-        alias="station"))  # List of stations information
+    stations: List[Station] = field(metadata=field_options(alias="station"))  # List of stations information
+
+
+@dataclass
+class Departures(DataClassORJSONMixin):
```

Ruff (0.8.2)

Missing docstring in public class

(D101)